### ISSUE TITLE
I uncommented all  the canvas/ctx lines - context is null

### ISSUE THREAD

**Issue by mikron123** [spite/virtual-webcam#3]
I uncommented all  the canvas/ctx lines and context return null..

Any idea why? Thanks

**Comment by birdieFly** [comment]
Make sure you comment out anything that uses or constructs "this.renderer". It's not needed if you just want to use CanvasRenderingContext2D instead of WebGL.

**Comment by mikron123** [comment]
Yeah I noticed you need to do that! Thanks it works great! Awesome work!

**Comment by birdieFly** [comment]
Can't take credit, not my work. I'm just a grateful user.

**Comment by birdieFly** [comment]
Sorry to use this channel, but @mikron123 did by any chance try to draw an image on the canvas and it is passed to the video? Whatever i try an image won't work. Text works fine, but image won't.

**Comment by mikron123** [comment]
@birdieFly  For you drawing Image on canvas question - there is a security restriction so it is blocked (as you probably see in the console error messages) I didn't get it to work on localhost, but did manage to load an image which is stored on the same domain as the website. if the image is on a storage service of some sort, you'll need to add the domain to the Cors AllowedOrigin settings..
